### ISSUE TITLE
Falcon lsf job Exited with signal termination 

### ISSUE THREAD

**Issue by ishengtsai** [PacificBiosciences/FALCON-integrate#94]
This is with 1.7.6 that the pipeline failed with several jobs. The data worked on local option.

cd /work1/r07ijt00/Pacbio/falcon/Aphe/mypwatcher/jobs/
`grep Exit */*
J1744346edb1387c55cce7afb0050342daf80521b129465eedfa55c452ec3b0db/stdout:Subject: Job 553065: <J1744346edb1387> in cluster <cluster5> Exited
J1744346edb1387c55cce7afb0050342daf80521b129465eedfa55c452ec3b0db/stdout:Exited with signal termination: Alarm clock.
J237c786c5c045546efbd3790ed5a93b9e78df568fcd4aa39a49f5261e1beab71/stdout:Subject: Job 553069: <J237c786c5c0455> in cluster <cluster5> Exited
J237c786c5c045546efbd3790ed5a93b9e78df568fcd4aa39a49f5261e1beab71/stdout:Exited with exit code 143.
J30c022c10e2aed0c7b7a9aff7f7baf30887cb4448754a7a351b8dce28034e663/stdout:Subject: Job 553141: <J30c022c10e2aed> in cluster <cluster5> Exited
J30c022c10e2aed0c7b7a9aff7f7baf30887cb4448754a7a351b8dce28034e663/stdout:Exited with exit code 143.
J34abcdd87885182a5cfd720fde3c55d27c6960693b67229222b8ccc2a658f935/stdout:Subject: Job 553061: <J34abcdd8788518> in cluster <cluster5> Exited
J34abcdd87885182a5cfd720fde3c55d27c6960693b67229222b8ccc2a658f935/stdout:Exited with signal termination: Alarm clock.`

The working directory seems executing ok? I also attach the stdout and stderr for your convenience. 
![image](https://cloud.githubusercontent.com/assets/5182142/20083632/86875242-a598-11e6-9674-962472e64537.png)

https://www.dropbox.com/sh/12sn97pd4wo3oty/AACd4ddQ0lcyOxAPmxubx8jna?dl=0

**Comment by pb-cdunn** [comment]
(Thanks for including stderr.)

The job seems fine. **LA4Falcon** passed. **fc_consensus** was running properly with 6 sub-processes, until it received **SIGINT** (Ctrl-C/KeyboardInterrupt). Apparently LSF killed these jobs. "Alarm clock"? Maybe your LSF limits the runtime. You can set extra LSF flags via `sge_option_cns` (iirc).

**Comment by ishengtsai** [comment]
Hi C,

We reverted it back to v.0.7.3 and the LSF worked fine (with the same cfg file). So was there any changes? I could test more version if you want.

Best,
Jason

**Comment by pb-cdunn** [comment]
Yes, there are changes, but the latest should work for you. I would be very helpful if you could learn what fails. Maybe try issuing the job-submission command again yourself, from the command-line. Also, compare the job-submission commands for the consensus step from 0.7.3 and from latest. There must be a difference somewhere. Only someone with LSF can debug this.
